data/strings/strings.txt

```diff
@@ -4323,6 +4416,7 @@
     ko = 죄송합니다, 지도 스토리지 설정이 비활성화되어 있습니다.
     lt = Atsiprašome, šiuo metu žemėlapio saugyklos nustatymai yra išjungti.
     mr = क्षमस्व, नकाशा साठवायचे सेटिंग सध्या बंद आहेत.
+    mt = Jiddispjaċini, is-settings tal-Ħażna tal-Mappa huma mitfija bħalissa. 
```

Comment:
Validation script complains about this line (see Checks). Is there a Windows line-ending (CR LF) instead of Unix (LF)?

Reply:
I think there was an extra white space, if you select my line you can see that, should be fixed now sorry.